### ISSUE TITLE
Results : GCN

### ISSUE THREAD

**Issue by BrandonN24** [BrandonN24/GCN-GAT-ComparativeAnalysis#11]
Report training and inference results for analysis for the GCN, adjust hyperparameters as needed. Make sure to document any changes to hyperparameters.
- Make note of parameters and hyperparameters used.
- Report training accuracy and loss across epochs
- Report confusion matrix
- Report accuracy metrics
- Report visual of true label graph vs. predicted label graph
